DESCRIPTION

```diff
@@ -57,7 +57,7 @@ Config/testthat/edition: 3
 Encoding: UTF-8
 Language: en-gb
 Roxygen: list(markdown = TRUE)
```

Comment:
Might want to go ahead and put a Remote on dev tidyr.

Also, while it probably won't make much of a difference, it might be best to change from Suggesting just `tidyr` to `tidyr (>= <the-current-dev-version>)`.

When tidyr is on CRAN we can remove the Remote and bump to the new CRAN version, since that will be required for this all to work right

Reply:
👍